src/bootstrap/doc.rs

```diff
@@ -162,7 +162,7 @@ pub fn std(build: &Build, stage: u32, target: &str) {
         cargo.arg("-p").arg("std");
     } else {
         cargo.arg("--no-deps");
-        for krate in &["alloc", "collections", "core", "std", "std_unicode"] {
+        for krate in &["alloc", "collections", "core", "std", "std_unicode", "proc_macro"] {
```

Comment:
Shouldn't this be a const near the top of the file instead of a magic number / magic slice?